kong/runloop/plugin_servers/pb_rpc.lua

```diff
@@ -371,7 +371,7 @@ function Rpc:call_start_instance(plugin_name, conf)
     return nil, err
   end
 
-  kong.log.debug("started plugin server: seq ", conf.__seq__, ", worker ", ngx.worker.id() or -1, ", instance id ",
+  kong.log.debug("started plugin server: seq ", conf.__seq__, ", worker ", ngx.worker.id(), ", instance id ",
```

Comment:
Please keep the `or -1` here, as `ngx.worker.id()` returns `nil` for privileged workers. We've changed all parts of the codebase, so please be consistent.

Ref: https://github.com/Kong/kong/pull/12385